test/unit/mixins/evented.test.js

```diff
@@ -61,33 +62,76 @@ QUnit.test('evented() with custom element', function(assert) {
   );
 });
 
+QUnit.test('trigger() errors', function(assert) {
```

Comment:
We never had a test for this before (since we didn't have an errors). I added it.